playbooks/common/openshift-glusterfs/config.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- name: Open firewall ports for GlusterFS
-  hosts: oo_glusterfs_to_config
+- name: Open firewall ports for GlusterFS nodes
+  hosts: glusterfs
```

Comment:
I'm accounting for the possibility that both `[glusterfs]` and `[gluster_registry]` are specified but that one is native while the other is external.